l1-contracts/test/sparta/Sparta.t.sol

```diff
@@ -180,6 +204,40 @@ contract SpartaTest is DecoderBase {
     _testBlock("mixed_block_2", false, 3, false);
   }
 
+  function testNukeFromOrbit() public setup(4) {
+    // We propose some blocks, and have a bunch of validators attest to them.
+    // Then we slash EVERYONE that was in the committees because the epoch never
```

Comment:
I don't think this is predicated on the epoch never finalizing?

Reply:
No not predicated, they could decide to slash for whatever. It was more a "story" around the test. Showing slashing, and then using case of blocks without proofs as "never" being proven because the test does not proof it. If proving in this test could still punish them just because you don't like them.